src/app/pages/ArticlePage/PagePromoSections/TopStoriesSection/index.styles.jsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import {
+  GEL_SPACING,
+  GEL_SPACING_BORDER,
+  GEL_SPACING_DBL,
+} from '#app/legacy/gel-foundations/src/spacings';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+} from '#app/legacy/gel-foundations/src/breakpoints';
+import {
+  StoryPromoUlBase,
+  StoryPromoLiBase,
+} from '#components/OptimoPromos/Lists';
+
+export const StyledWrapper = styled.section`
+  @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING};
+  }
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    padding: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+export const FlexPromoList = styled(StoryPromoUlBase)`
```

Comment:
The name here describes the styles, which can change, making the name confusing. Also, it's flex at a very specific breakpoint

Reply:
fixed